1.1/spec/17-Annex-B.adoc

```diff
@@ -323,12 +321,11 @@ eg:x
         geo:isEmpty false ;
         geo:isSimple true ;
         geo:hasSerialization "<http://www.opengis.net/def/crs/EPSG/0/4326> POLYGON ((149.060 -35.236, ... , 149.060 -35.236)))"^^geo:wktLiteral ;
-        geo:inSRS <http://www.opengis.net/def/crs/EPSG/0/4326> ;
     ] ;
 . 
 ```
 
-In this example, each of the standards properties defined for a `Geometry` instance has realistic values, for example, the http://www.opengis.net/ont/geosparql#isempty[`geo:isEmpty`] is set to `false` since the geometry contains information. Note that the SRS of the given serialzation is given within the geometry literal (a WKT value) as well as by the http://www.opengis.net/ont/geosparql#inSRS[`geo:inSRS`] property.
+In this example, each of the standards properties defined for a `Geometry` instance has realistic values, for example, the http://www.opengis.net/ont/geosparql#isempty[`geo:isEmpty`] is set to `false` since the geometry contains information.
```

Comment:
"geometry" should be "Geometry"

Reply:
Fixed as requested in commit https://github.com/opengeospatial/ogc-geosparql/pull/162/commits/e8fae58e396d5a1c9762cc55929e3ca0cfa17f9e